CHANGELOG.md

```diff
@@ -2,6 +2,103 @@
 
 All notable changes to this project will be documented in this file.
 
+## Unreleased - as of Sprint 47 end 2016-10-03
+
+### [Added](https://github.com/ManageIQ/manageiq/issues?q=milestone%3A%22Sprint+47+Ending+Oct+3%2C+2016%22+label%3Aenhancement)
+
+- Automate
+  - Provisioning
+    - Filter networks and floating_ips for OpenStack provisioning
+    - Added Google pre and post provisioning methods
+    - Enable support for vApp provisioning service
+    - Allow VMware provision without needing to specify a host
+    - Set VM storage profile in provisioning
+  - Services: Log properly dynamic dialog field script error
+  - Modeling to support generic object
```

Comment:
Along with this include PR #10922: "Process Generic Object method call via automate"